src/core/thread/mle.hpp

```diff
@@ -745,6 +745,17 @@ class Mle : public InstanceLocator, private NonCopyable
      *
      */
     void SetCslTimeout(uint32_t aTimeout);
+
+    /**
+     * This method calculates CSL metric of parent.
```

Comment:
Maybe extend the docs here to explain what metric is being used to compare two parents?

Reply:
Fixed https://github.com/openthread/openthread/pull/6802/files#diff-47bc8c061f3aab7ae9b35d5bf7ab39d53cf6def0647d24fdf24b6e6b99ed415cR4512